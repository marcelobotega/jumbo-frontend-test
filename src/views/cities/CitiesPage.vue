<template>
  <div class="cities-page-wrapper">
    <CitiesList />
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
import CitiesList from "@/components/cities-list/CitiesList.vue";

@Component({
  components: {
    CitiesList,
  },
})
export default class CitiesPage extends Vue {}
</script>
