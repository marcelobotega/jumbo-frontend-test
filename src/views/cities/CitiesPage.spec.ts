import { shallowMount } from "@vue/test-utils";
import CitiesPage from "@/views/cities/CitiesPage.vue";

describe("CitiesPage.vue", () => {
  it("should create", () => {
    const wrapper = shallowMount(CitiesPage);
    expect(wrapper).toMatchSnapshot();
  });
});
