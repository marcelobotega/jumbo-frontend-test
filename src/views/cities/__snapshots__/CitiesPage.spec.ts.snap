// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesPage.vue should create 1`] = `
<div class="cities-page-wrapper">
  <citieslist-stub></citieslist-stub>
</div>
`;
